src/main/java/uk/gov/hmcts/divorce/common/event/SubmitAos.java

```diff
@@ -96,11 +117,35 @@ private PageBuilder addEventConfig(ConfigBuilder<CaseData, State, UserRole> conf
             .description("Submit AoS")
             .showSummary()
             .aboutToSubmitCallback(this::aboutToSubmit)
+            .submittedCallback(this::submitted)
             .explicitGrants()
             .grant(CREATE_READ_UPDATE, APPLICANT_2_SOLICITOR, APPLICANT_2)
             .grant(READ,
                 CASE_WORKER,
                 LEGAL_ADVISOR,
                 SUPER_USER));
     }
+
+    public SubmittedCallbackResponse submitted(final CaseDetails<CaseData, State> details,
+                                               final CaseDetails<CaseData, State> beforeDetails) {
+
+        log.info("Caseworker submit aos submitted callback invoked for case id: {}", details.getId());
+
+        final AcknowledgementOfService acknowledgementOfService = details.getData().getAcknowledgementOfService();
+
+        String eventId;
```

Comment:
personally I would have used a ternary statement here but that's just a preference thing